docs/en_us/platform_api/Makefile

```diff
@@ -1,164 +0,0 @@
-# Makefile for Sphinx documentation
```

Comment:
I'm nervous about removing the platform_api directory as I believe that is still used to generate:

https://edx.readthedocs.io/projects/edx-platform-api/en/latest/

I'd suggest that you don't remove this directory. I also think that you should ensure that the doc can still build with your changes (for example, I think you might need some of the requirements files).

@catong am I correct that this document is still active?